src/app/routes/index.test.jsx

```diff
@@ -67,7 +69,7 @@ const getMatchingRoute = pathname =>
 
 const renderRouter = props =>
   act(async () => {
-    await render(
+    render(
```

Comment:
^^ is that deliberate?

Reply:
Yes, the linter suggested that the `await` had no affect in this context.